config/enterprise_versions.yml

```diff
@@ -15,6 +15,9 @@ components:
   cnx-queryserver:
     image: tigera/cnx-queryserver
     version: master
+  goldmane:
```

Comment:
Should we just remove this from enterprise then?

Reply:
done.